test/fixtures/parser/snowflake/snowflake_escape.yml

```diff
@@ -2,15 +2,15 @@ file:
   statement:
     select_statement:
       select_clause:
-        - keyword: select
-        - select_clause_element:
-            literal: "'c\\' '"
-            alias_expression:
-              keyword: as
-              identifier: escaped
-        - comma: ','
-        - select_clause_element:
-            literal: "'c\\' '' '"
-            alias_expression:
-              keyword: as
-              identifier: escaped_double
```

Comment:
Again, is it confusing to add these unrelated changes to this PR?

Reply:
This is another of what I described above -- unintentional changes introduced by the script. A nuisance for sure, but the right fix is to fix the script so the output is consistently formatted.